src/main/java/by/georgprog/epicmusicstore/mapper/UserMapper.java

```diff
@@ -0,0 +1,16 @@
+package by.georgprog.epicmusicstore.mapper;
+
+import by.georgprog.epicmusicstore.dto.UserDto;
+import by.georgprog.epicmusicstore.model.user.UserEntity;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
```

Comment:
Use `@Mapper(componentModel = "spring")` then you may inject mappers as normal Spring beans and call them by the same way as services like `userMapper.toDto(userEntity);` without any shit